dist/components/Header/Header.js

```diff
@@ -192,13 +192,13 @@ var Header = function (_React$Component) {
       _dgxFeatureFlags2.default.store.listen(this.onFeatureFlagsChange.bind(this));
       // Set the log out link to state
       this.setLogOutLink(window.location.href);
+      // Check if the cookie "PAT_LOGGED_IN" exists and then set the timer for deleting it
+      _encoreLogOutTimer2.default.setEncoreLoggedInTimer(window.location, this.state.currentTime);
```

Comment:
What if `window.location` is `"https://browse.nypl.org/iii/encore/record/C__Rb20970375__Stoast__P0,2__Orightresult__U__X7?lang=eng"`? Based on what `setEncoreLoggedInTimer` does with the location you give it, do you actually want to use `window.location.host`?

Reply:
The real function that uses `window.location.href` is `renderDynamicLogOutLink` in `src/utils`. It uses `window.location.href` as the value of `redirect_uri` for the log out link so we will need the whole URI.

Reply:
Sorry - I'm still missing something. `setEncoreLoggedInTimer` only seems to set `ENCORE_LAST_VISITED` if `currentLocation` is `"'browse.nypl.org'"`, and above you're setting `currentLocation` to `window.location`, which will never equal `"browse.nypl.org"`. So I don't see how `ENCORE_LAST_VISITED` can ever be set

Reply:
I see what you are saying. They are actually two different mechanisms.

The function `this.setLogOutLink(window.location.href)` is to set the log out link in the dropdown menu on Header. It is not related to the iframe from this PR. You can see the log out link if you log in and click my account on Header. The link will be the URL of the log out button.

While `setEncoreLoggedInTimer` is for automatically logging the user out by setting a timer, and timer is based on when is the last time the user has visited the page with the domain of `browse.nypl.org`.